react/components/UtilityFunctionsPixexEvents.js

```diff
@@ -0,0 +1,21 @@
+export const getCategoryFromObjs = (products) => {
+  if (products[0] === null || products[0] === undefined) {
+    return ''
+  }
+
+  const { categoryId } = products[0]
+  const result = products.every((product) => product.categoryId === categoryId)
+
+  return result ? categoryId : ''
+}
+
+export const pushPixelEvent = (name, value, products, push) => {
```

Comment:
I think `pushFilterManipulationPixelEvent` is more descriptive, even if it's longer. `pushPixelEvent` gives the impression that it could be used to push any pixel event, which is not the case.

Reply:
I agree with you